README.md

```diff
@@ -2,3 +2,136 @@
 
 This document explains our approach to working with Rubocop in dxw. We should
 in general:
+
+### Basics
+- use the latest version of rubocop
+- configure CI to run rubocop and fail if there are *any* violations
+- configure your editor to quickly and loudly complain about rubocop
+  violations
+  - otherwise it'll fail on CI and you'll feel silly
+  - or it'll fail too late and you'll end up re-writing code to make it pass
+- use the `.rubocop.yml` file in this repo as a starting point.
+- keep the `.rubocop.yml` file in this repo up-to-date with the latest version
+  of rubocop
+
+### Exceptions
+- avoid adding additional exceptions to the defaults unless there's a really
+  good reason.
+  - Even if we don't personally like some of the decisions there's value in
+    keeping the number of exceptions to the defaults to a minimum.
+- consider if any rules we do add should also be added to the default config
+  in this repo
+  - Differences between projects should be kept to a minimum.
+
+### Todos
+- avoid creating a `.rubocop_todo.yml` file for new projects
```

Comment:
I think this is unlikely to happen, as we won't be able to create a `rubocop_todo` until we've written some code that violates the rules. If that code merits exceptions, then we're into `Handling exceptions to rules`, below.

Reply:
Got it: my wording was unclear: the intention was "this is OK for legacy projects which didn't start off with a rubocop_todo file, but not for projects which had rubocop from day one" - I'll clarify the text.

Reply:
Fixed as a separate commit (to be squashed in)